frontend/src/views/Administration.vue

```diff
@@ -16,89 +16,213 @@ limitations under the License.
-
-      <v-toolbar class="red elevation-0 darken-2" dark>
-        <v-icon class="white--text pr-2">mdi-cube</v-icon>
-        <v-toolbar-title>Project Details</v-toolbar-title>
-        <v-spacer></v-spacer>
-        <v-tooltip v-if="canDeleteProject" top>
-          <template v-slot:activator="{ on }">
-            <div v-on="on">
-              <v-btn :disabled="isDeleteButtonDisabled" icon @click.native.stop="showDialog">
-                <v-icon>delete</v-icon>
-              </v-btn>
-            </div>
-          </template>
-          <span v-text="isDeleteButtonDisabled ? 'You can only delete projects that do not contain clusters' : 'Delete Project'" />
-        </v-tooltip>
-      </v-toolbar>
-
-      <v-card-text>
-        <v-row >
-          <v-col lg="4" cols="12">
-            <label class="caption grey--text text--darken-2">Name</label>
-            <p class="subtitle-1">{{projectName}}</p>
-          </v-col>
-          <v-col lg="4" cols="12">
-            <label class="caption grey--text text--darken-2">Technical Contact</label>
-            <p class="subtitle-1"><account-avatar :account-name="technicalContact" :mail-to="true"></account-avatar></p>
-          </v-col>
-
-        </v-row>
-        <v-row >
-          <v-col lg="4" cols="12">
-            <label class="caption grey--text text--darken-2">Created At</label>
-            <p>
-              <v-tooltip right>
-                <template v-slot:activator="{ on }">
-                  <span v-on="on" class="subtitle-1">{{createdAt}}</span>
+    <v-row no-gutters>
+      <v-col cols="12" md="6">
+        <v-row no-gutters class="flex-column">
+          <v-col class="pa-3">
+            <v-card>
+              <v-toolbar flat dark dense :color="color">
+                <v-toolbar-title class="subtitle-1">Details</v-toolbar-title>
+              </v-toolbar>
+              <v-list>
+                <v-list-item>
+                  <v-list-item-avatar>
+                    <v-icon :color="color">info_outline</v-icon>
+                  </v-list-item-avatar>
+                  <v-list-item-content>
+                    <v-list-item-subtitle>Name</v-list-item-subtitle>
+                    <v-list-item-title>
+                      {{projectName}}
+                    </v-list-item-title>
+                  </v-list-item-content>
+                  <v-list-item-action>
+                    <copy-btn :color="color" :clipboard-text="projectName" tooltipText="Copy project name to clipboard"></copy-btn>
+                  </v-list-item-action>
+                </v-list-item>
+                <v-list-item>
+                  <v-list-item-avatar>
+                    <v-icon :color="color">mdi-text-subject</v-icon>
+                  </v-list-item-avatar>
+                  <v-list-item-content>
+                    <v-list-item-subtitle>Description</v-list-item-subtitle>
+                    <v-list-item-title class="wrap-text">
+                      <editable-text
```

Comment:
for all edit actions: check for `canPatchProject`

Reply:
✔️